internal/cli/atlas/deployments/delete.go

```diff
@@ -42,11 +42,7 @@ func (opts *DeleteOpts) Run(ctx context.Context) error {
 	}
 
 	telemetry.AppendOption(telemetry.WithDeploymentType(options.LocalCluster)) // always local
-	if opts.DeploymentName != "" {
-		if err := opts.DeploymentOpts.CheckIfDeploymentExists(ctx); err != nil {
-			return err
-		}
-	} else {
+	if opts.DeploymentName == "" {
```

Comment:
removing the check because we can have left overs

Reply:
if we find no left overs, shouldn't we still say "deployment not found? "